packages/sdk-react/CHANGES.md

```diff
@@ -1,5 +1,9 @@
 fusionauth-react-sdk Changes
 
+Changes in 2.4.1
+
+- Fix the bug where the user is logged out on a successful token refresh.
+
```

Comment:
Resolved in PR [161](https://github.com/FusionAuth/fusionauth-javascript-sdk/pull/161)